prow/manifests/overlays/metal3/config.yaml

```diff
@@ -2735,6 +2735,11 @@ presubmits:
     agent: jenkins
     always_run: false
     optional: true
+  # {job_prefix}-ironic-image-build-test
+  - name: metal3-ironic-image-build-test
```

Comment:
Does this work equally in all Ironic branches?

Reply:
I think it is the same the old job https://gerrit.nordix.org/plugins/gitiles/infra/cicd/+/refs/heads/master/jjb/metal3/job_ironic_image_build_test.yml#82 was white listed on all the supported branches

Reply:
The white list is missing release-25.0 btw.